packages/gluestick-cli/cli.js

```diff
@@ -66,27 +68,68 @@ commander
     } catch (error) {
       exitWithError('Invalid package.json');
     }
-    const wmlBin = path.join(__dirname, './node_modules/.bin/wml');
-    spawn.sync(
-      wmlBin,
-      ['rm', 'all'],
-      { stdio: 'inherit' },
-    );
-    spawn.sync(
-      wmlBin,
-      ['add', srcPath, path.join(process.cwd(), 'node_modules/gluestick')],
-      { stdio: 'inherit' },
-    );
-    spawn.sync(
-      'watchman',
-      ['watch', srcPath],
-      { stdio: 'inherit' },
-    );
-    spawn.sync(
-      wmlBin,
-      ['start'],
-      { stdio: 'inherit' },
-    );
+    const convertFilePath = filePath => filePath.replace(srcPath, path.join(process.cwd(), 'node_modules/gluestick'));
+    const watcher = chokidar.watch(`${srcPath}/**/*`, {
+      ignored: [
+        /(^|[/\\])\../,
+        /.*node_modules.*/,
+      ],
+      persistent: true,
+    });
+    watcher
+      .on('ready', () => {
+        console.log(chalk.blue('Watching for changes...'));
+        watcher.on('add', filePath => {
```

Comment:
This event handler should be separate function because handler from `change` event look almost the same, so this function should accept argument eg.`type` that will be used in console.log like `added` or `changed`.